app/patients/photo/controller.js

```diff
@@ -15,7 +15,7 @@ export default AbstractEditController.extend({
   showUpdateButton: true,
 
   database: inject.service(),
-  editController: inject.controller('patients/edit'),
+  editController: null,
```

Comment:
This line should be removed as editController is not used from this context anymore.